atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -144,11 +145,14 @@ private InetSocketAddress getAddressForHostThrowUnchecked(String host) {
     }
 
     public InetSocketAddress getAddressForHost(String host) throws UnknownHostException {
-        InetAddress resolvedHost = InetAddress.getByName(host);
+        if (config.addressTranslation().containsKey(host)) {
```

Comment:
1) Will describe_ring always give us back the same addresses for the other nodes?  Is there any variation in format?
We may have to put multiple entries for the same host into the map, not a problem just noting it.

2) What if a data centre rack is replaced and the C* node is moved, will the address remain the same?
I'm trying to think up scenarios where the config might have to be dynamic.  May not be needed but I want to be sure.